### PR TITLE
(grc_conversion) EpiDoc and CTS conversion 

### PR DESCRIPTION
**Pull request by lcerrato**
#1399 continues with tlg 0086 batch

### CODE REVIEW

File: data/tlg0086/tlg010/__cts__.xml
Comment (AlisonBabeu) [inline data/tlg0086/tlg010/__cts__.xml:1]: I think that should be Bywater, Ingram!

File: data/tlg0086/tlg029/tlg0086.tlg029.perseus-grc2.xml
Comment (AlisonBabeu) [inline data/tlg0086/tlg029/tlg0086.tlg029.perseus-grc2.xml:1]: So I noticed there are two structured bibls in this file, including this second one:

           `  <biblStruct>` 
              `  <monogr>` 
                 `  <author>Aristotle</author>` 
                  ` <title>Aristotle in 23 Volumes, Vol. 18</title>` 
                 `  <imprint>` 
                     ` <publisher>Harvard University Press, Cambridge, Mass., London</publisher>` 
                  `    <date>1935</date>` 
                 `  </imprint>` 
            `    </monogr>` 
           `  </biblStruct>`

I was wondering if this second couldn't be captured instead in a series statement, or if this was just left by accident.

**Review by AlisonBabeu — CHANGES_REQUESTED** [review]
hi @lcerrato just one or two changes!

File: data/tlg0086/tlg029/tlg0086.tlg029.perseus-grc2.xml
Comment (lcerrato) [reply data/tlg0086/tlg029/tlg0086.tlg029.perseus-grc2.xml:1]: That was the original that should be deleted.

**Pull request merged**